var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -32,8 +32,10 @@ class Relion(CMakePackage):
     electron cryo-microscopy (cryo-EM)."""
 
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
-    url      = "https://github.com/3dem/relion/archive/2.0.3.tar.gz"
+    url      = "https://github.com/3dem/relion"
 
+    version('2.1', git='https://github.com/3dem/relion.git',
+    commit='e7607a869687b636d3c39e0d5b6a9cba930fc3b2')
```

Comment:
Why this commit?

Reply:
It's the latest.  In the past I was instructed to point the latest commit though I could rather set it to master if people prefer that instead.  This way we could always pull the latest fixes.

Reply:
The entire program is beta though if this is such a big deal then I can change the version name.

Reply:
@davydden, comments?

Reply:
i would name it `develop` and point to the main branch.